### ISSUE TITLE
Separating dispute resolution from CoC violations

### ISSUE THREAD

**Issue by dbooth-boston** [w3c/PWETF#348]
Reviewing the [Dispute Resolution document](https://github.com/w3c/PWETF/blob/main/DisputeResolution.md).

The title of the Dispute Resolution document, and some of its sections, give the impression that it is intended to address a disagreement among equals.   But portions of it discuss the process for addressing a CoC violation -- filing a complaint -- which to my mind is very different.  Furthermore, the CoC points to another document called [Positive Work Environment](https://www.w3.org/about/positive-work-environment/#Procedures) that supposedly describes the procedure for addressing a CoC violation, although the description given is currently so minimal that it really doesn't.

I think it would be better to separate dispute resolution from addressing a CoC violation:

- Make the Dispute Resolution document only be about resolving disagreements between parties -- not addressing CoC violations -- though it should reference how to file a complaint for a CoC violation.
- Move material about CoC violations that is currently in the Dispute Resolution document to whatever document is responsible for defining the procedures for addressing CoC violations -- currently the [Positive Work Environment](https://www.w3.org/about/positive-work-environment/#Procedures) document?
-

**Comment by swickr** [comment]
"Steps to Progressive Resolution of Conflict"?

**Comment by dbooth-boston** [comment]
I think it's belittling to frame a CoC violation as merely a "dispute" or a "conflict".  I think it is important that this document be clearly about how to handle CoC violations -- not disagreements between parties.

In our last meeting, Wendy explained that the current text was modeled after union dispute resolution language, and that probably explains why those words were used.  But a CoC violation is different. I think the title and framing should be adjusted to reflect this difference.

Title suggestions: "Steps to Resolve CoC Violations" or "Resolving CoC Violations"

**Comment by nigelmegitt** [comment]
> how to handle CoC violations -- not disagreements between parties

There may be some cases where all parties agree that there was a CoC violation, including the person who violated the CoC, but it is likely often to be the case ones that there is effectively an "accused" and an "accuser", who do not agree with each other. So two steps are needed:

1. Independently assess whether a CoC violation occurred
2. Take appropriate action if a CoC violation did occur

**Comment by dbooth-boston** [comment]
Agreed.  It's an alleged violation until it has been determined to have been an actual violation.  (And incidentally, determination of an actual violation does NOT depend on all parties reaching agreement.  That's one reason why it should not be framed as a "dispute".)  But the fact that a case starts off as an alleged violation doesn't change the fact that this document should be about resolving alleged or actual violations, rather than resolving "disputes".
